backend/btrixcloud/storages.py

```diff
@@ -405,7 +406,10 @@ async def get_next_chunk(file_, min_size) -> bytes:
             key += filename
 
             mup_resp = await client.create_multipart_upload(
-                ACL="bucket-owner-full-control", Bucket=bucket, Key=key
+                ACL="bucket-owner-full-control",
+                Bucket=bucket,
+                Key=key,
+                ContentType=mime or "",
```

Comment:
Would it maybe be worth setting mime based on file extension, rather than just whatever is sent from users of this method, especially if it's an optional param?

Reply:
See above, yeah, I think we're computing it from the mime type, and storing it, and then here just using that value